pacoloco.go

```diff
@@ -315,14 +398,21 @@ func handleRequest(w http.ResponseWriter, req *http.Request) error {
 // downloadFileAndSend downloads file from `url`, saves it to the given `localFileName`
 // file and sends to `clientWriter` at the same time.
 // The function returns whether the function sent the data to client and error if one occurred
-func downloadFile(url string, filePath string, ifModifiedSince time.Time, clientWriter http.ResponseWriter) (downloaded bool, err error) {
+func downloadFile(upstream_url string, filePath string, ifModifiedSince time.Time, clientWriter http.ResponseWriter) (downloaded bool, err error) {
```

Comment:
`upstream_url` should be named as `upstreamUrl` to match the standard naming convention.